### ISSUE TITLE
Cached movie on RD but message "No debrid result"

### ISSUE THREAD

**Issue by DaisyF8** [Sam-Max/plugin.video.jacktook#2]
I'm using RD, prowlarr on docker, and I'm crosschecking the results JackTook gives with the results I get manually in Prowlarr.  I can see in events that JT is communicating with Prowlarr no problem.

I'm having a very limited amount (almost never) of cached results for movies.  When I manually take a result from JT (show uncached results) and enter it in RD the movie is cached.

I'm willing to do tests for you for my setup.

EDIT: Kodi 21 Beta 2

**Comment by Sam-Max** [comment]
It should be because trackers that you are using on prowlarr give an url of a torrent file and not a magnet link, and JT failed to extract the infohash from the torrent file.  RD needs the infohash to check if it is cached. I need to improve that infohash extraction from torrent files. And how do you manually take a result on JT and enter it in RD?

**Comment by Sam-Max** [comment]
If you have Telegram write me here https://t.me/sammax09, to give me more detailed information about your prowlarr setup, to help me improve it.

**Comment by DaisyF8** [comment]
(EDIT: Total edit of what was here initially,  I found some stuff out and rewrote it.)

I don't take a result out of JT. I manually do the same search in Prowlar using the searchstring JT sent to Prowlarr that I see in Prowlarrs events. The number of results for my manual search is 99% the same as I see in JT notification in Kodi.  After I do a manual search in Prowlar I go to the results-urls, copy the magnet from the trackersite, and check/enter it in RD-website torrent tab. Low tech I know.

I'm limiting JT to only use a tracker (btmed_dot_c_o_m) that delivers magnet results.  When I check RD after I did the search in JT for for example a famous superhero movie starting a marvel franchise (resulting in 3 results in Prowlar) I see it added one magnet and is waiting for the file selection for two others.  JT didn't just check for cache, it added the three results.
